### PR TITLE
List function aliases in a more compact layout

### PR DESCRIPTION
**Pull request by seisman**
As shown in the screenshot below (the left one), currently, the function aliases are listed vertically, wasting a lot of vertical spaces. This PR uses the [`hlist`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-hlist) directive to make the list more compact, by listing 4 items per row (the right one).

| [The main branch](https://www.pygmt.org/dev/api/generated/pygmt.Figure.basemap.html) | [This PR](https://pygmt-dev--3666.org.readthedocs.build/en/3666/api/generated/pygmt.Figure.basemap.html) |
|---|---|
| image | image |

**Preview**: https://pygmt-dev--3666.org.readthedocs.build/en/3666/api/generated/pygmt.Figure.basemap.html

### CODE REVIEW

**Comment by weiji14** [comment]
Could you try 3 columns instead of 4? The columns seem to go too close to the edge of the browser window on a less wide window (e.g. when I have two windows side by side).

image

File: pygmt/helpers/decorators.py [hunk 1]
```diff
@@ -456,18 +456,22 @@ def fmt_docstring(module_func):
...
+       - J = projection
+       - R = region
     <BLANKLINE>
     """  # noqa: D410,D411
     filler_text = {}
 
     if hasattr(module_func, "aliases"):
         aliases = ["**Aliases:**\n"]
+        aliases.append(".. hlist::")
+        aliases.append("   :columns: 4\n")
```
Comment (seisman) [inline pygmt/helpers/decorators.py:471]:
```suggestion
        aliases.append("   :columns: 3\n")
```

**Comment by weiji14** [comment]
Readthedocs build failing because of https://github.com/GenericMappingTools/pygmt/issues/3301#issuecomment-2513054819 :sweat_smile: Docs build isn't broken yet, probably due to caching, but it'll probably break on newer PRs.

**Comment by michaelgrund** [comment]
With 3 columns it looks good to me. Makes the whole thing also better readable in my opinion since you dont have to scroll down for long lists.

**Review by weiji14 — APPROVED** [review]
Yep, looks better now on half a screen!

image

**Pull request merged**